### PR TITLE
[Doc/Serve] Add Arize integration

### PR DESCRIPTION
**Pull request by sihanwang41**
Signed-off-by: Sihan Wang <sihanwang41@gmail.com>

## Why are these changes needed?

Add a section helping the user to find the integration code of Arize.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/serve/production-guide/monitoring.md [hunk 1]
```diff
@@ -300,3 +301,10 @@ ray_my_counter{..., deployment="MyDeployment"} 5.0
 ```
 
 See the [Ray Metrics documentation](ray-metrics) for more details, including instructions for scraping these metrics using Prometheus.
+
+## Exporting metrics into Arize
+Besides using Prometheus to check out Ray metrics, Ray Serve also has the flexibility to export the metrics into other observability platform.
```
Comment (architkulkarni) [inline doc/source/serve/production-guide/monitoring.md:306]:
```suggestion
Besides using Prometheus to check out Ray metrics, Ray Serve also has the flexibility to export the metrics into other observability platforms.
```

File: doc/source/serve/production-guide/monitoring.md [hunk 2]
```diff
@@ -300,3 +301,10 @@ ray_my_counter{..., deployment="MyDeployment"} 5.0
...
+
+[Arize](https://docs.arize.com/arize/) is the machine learning observability platform which can help you to monitor real-time model performance, root cause model failures/performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues etc.
+
+To integrate with Arize, you can directly add Arize client code into your serve deployment code. ([Example code](https://docs.arize.com/arize/integrations/integrations/anyscale-ray-serve))
```
Comment (architkulkarni) [inline doc/source/serve/production-guide/monitoring.md:310]:
```suggestion
To integrate with Arize, you can directly add Arize client code into your Serve deployment code. ([Example code](https://docs.arize.com/arize/integrations/integrations/anyscale-ray-serve))
```

File: doc/source/serve/production-guide/monitoring.md [hunk 3]
```diff
@@ -300,3 +301,10 @@ ray_my_counter{..., deployment="MyDeployment"} 5.0
 ```
 
 See the [Ray Metrics documentation](ray-metrics) for more details, including instructions for scraping these metrics using Prometheus.
+
+## Exporting metrics into Arize
+Besides using Prometheus to check out Ray metrics, Ray Serve also has the flexibility to export the metrics into other observability platform.
+
+[Arize](https://docs.arize.com/arize/) is the machine learning observability platform which can help you to monitor real-time model performance, root cause model failures/performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues etc.
```
Comment (simon-mo) [inline doc/source/serve/production-guide/monitoring.md:308]:
```suggestion
[Arize](https://docs.arize.com/arize/) is the machine learning observability platform which can help you to monitor real-time model performance, root cause model failures or performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues, and much more.
```

**Review by simon-mo — APPROVED** [review]

File: doc/source/serve/production-guide/monitoring.md (hunk 3 above)
Comment (architkulkarni) [inline doc/source/serve/production-guide/monitoring.md:308]:
```suggestion
[Arize](https://docs.arize.com/arize/) is a machine learning observability platform which can help you to monitor real-time model performance, root cause model failures/performance degradation using explainability & slice analysis and surface drift, data quality, data consistency issues etc.
```

**Review by architkulkarni — APPROVED** [review]

**Comment by architkulkarni** [comment]
Assigning @dmatrix as representative of ray-docs codeowners

**Pull request merged**